internal/cmd/logout.go

```diff
@@ -80,14 +80,13 @@ func logoutOfServer(hostConfig *ClientHostConfig) (success bool, err error) {
 	err = client.Logout()
 	switch {
 	case api.ErrorStatusCode(err) == http.StatusUnauthorized:
-		logging.S.Warn(err.Error())
-		return false, nil
+		return false
```

Comment:
we should add a debug log for this error too, could be hard to troubleshoot otherwise